deploy/operator/clickhouse-operator-install-template.yaml

```diff
@@ -2391,7 +2391,7 @@ spec:
       containers:
         - name: clickhouse-operator
           image: $OPERATOR_IMAGE
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
It will override after run `./deploy/operator/build-clickhouse-operator-install-yaml.sh`
so, please change './deploy/dev/clickhouse-operator-install-yaml-template-04-section-deployment.yaml`
and run 
```
bash -xe ./deploy/operator/build-clickhouse-operator-install-yaml.sh
git add *
git commit -m "update install template"
git push origin ref_tests
```